teddy.js

```diff
@@ -229,32 +227,13 @@
             }
           }
         }
-      } else {
+      } else {  
```

Comment:
Remove that whitespace. (honestly surprised standard didn't catch that)

Reply:
fixed this